cluster-autoscaler/Dockerfile

```diff
@@ -15,6 +15,9 @@
 FROM gcr.io/google_containers/ubuntu-slim:0.1
 MAINTAINER Marcin Wielgus "mwielgus@google.com"
 
+RUN apt-get update  
+RUN apt-get install -y ca-certificates
```

Comment:
nit: To save dozens of MB from the resulting docker image size, I'd suggest replacing these 2 lines with something like:
```
RUN apt-get update -y && \
  apt-get install -y --no-install-recommends ca-certificates && \
  apt-get clean -y && \
  rm -rf /var/cache/apt/archives/* /var/lib/apt/lists/*
```